File: Php/app/Urls/CashgameIndexUrlModel.php
<?php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class CashgameIndexUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::cashgameIndex, $homegame);
		}

	}

}

File: Web/Models/CashgameIndexUrlModel.cs
using Core.Classes;
using Web.Models;
using Web.Routing;

namespace app{

	public class CashgameIndexUrlModel : HomegameUrlModel{

		public CashgameIndexUrlModel(Homegame homegame) : base(RouteFormats.CashgameIndex, homegame){}

	}

}